deps/wrapper.cc

```diff
@@ -131,6 +183,44 @@ bool JuliaGcsClient::Exists(const std::string &ns,
     return exists;
 }
 
+ObjectID _submit_task(std::string project_dir) {
+    auto &worker = CoreWorkerProcess::GetCoreWorker();
+
+    RayFunction func(
+        Language::JULIA,
+        FunctionDescriptorBuilder::BuildJulia("", "demo_task", "")
+    );
+
+    std::vector<std::unique_ptr<TaskArg>> args;
+    std::string str = "hello";
+    auto buffer = std::make_shared<LocalMemoryBuffer>(reinterpret_cast<uint8_t *>(&str[0]), str.size(), true);
+    auto ray_obj = std::make_shared<RayObject>(buffer, nullptr, std::vector<rpc::ObjectReference>());
+    args.emplace_back(new TaskArgByValue(ray_obj));
```

Comment:
we're not using these args right now right?  the callback function just creates a ray object from the return value of `f` (the PID) and nothign else AFAICT